spec/support/api/helpers.rb

```diff
@@ -21,19 +21,11 @@ def init_api_spec_env
 
         def api_basic_authorize(*identifiers, user: @user.userid, password: @user.password)
           if identifiers.present?
-            identifiers.flatten.collect do |identifier|
-              if identifier
-                # TODO:  We create product features on demand for various tests instead of seeding the correct features.
-                # We shouldn't hardcode a feature_type here but since it's a UI concern and it's now required, it shouldn't affect the API.
-                # See also: https://github.com/ManageIQ/manageiq/pull/21207
-                feature = MiqProductFeature.find_by(:identifier => identifier)
-                feature ||= MiqProductFeature.create(:identifier => identifier, :feature_type => "view")
-                @role.miq_product_features << feature
-              end
-            end
-            @role.save
-
-            MiqProductFeature.seed_tenant_miq_product_features if identifiers & MiqProductFeature::TENANT_FEATURE_ROOT_IDENTIFIERS == identifiers
+            identifiers = identifiers.flatten
```

Comment:
Not sure this is needed, but it was in the original code, so I kept it.

Reply:
Ok, it's needed - seems the helper methods that return a set of features from the api.yml pass them as an array, and thus `*identifiers` becomes an array of arrays.